src/PageLayout/PageLayout.tsx

```diff
@@ -178,18 +179,22 @@ const VerticalDivider: React.FC<DividerProps> = ({variant = 'none', variantWhenN
 export type PageLayoutHeaderProps = {
   divider?: 'none' | 'line'
   dividerWhenNarrow?: 'inherit' | 'none' | 'line' | 'filled'
```

Comment:
Non-blocking nitpick because it's unrelated to this PR: I find it a little strange that this type is different from `divider`. I'm do we never want `'filled'` on wider viewports?

Reply:
I'll defer to @vdepizzol